src/pages/api/auth/[...nextauth].ts

```diff
@@ -2,6 +2,46 @@ import OktaProvider from 'next-auth/providers/okta';
 import type { NextAuthOptions } from 'next-auth';
 import type { NextApiRequest, NextApiResponse } from 'next';
 import NextAuth from 'next-auth';
+import * as Sentry from '@sentry/nextjs';
+
+async function getUser(userId: string): User | null {
+    const query = 'userId=' + userId;
+    const url = `${process.env.PERSONALIZATION_URL}/user_preferences?${query}`;
+    const options = {
+        method: 'GET',
```

Comment:
`GET` is the default method so I don't think it needs to be explicitly specified here.